src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,154 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        Assertions.assertThrows(RegistrationDataException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void register_nullUserLogin_notOk() {
+        User user = new User();
```

Comment:
As you create user several times the same way pls extract user creation to a separate method to avoid repeating.
Method can take login, password and age as parameters.